exporter/awsxrayexporter/translator/http_test.go

```diff
@@ -264,6 +264,26 @@ func TestHttpStatusFromSpanStatus(t *testing.T) {
 	assert.True(t, strings.Contains(jsonStr, "200"))
 }
 
+func TestSpanWithNotEnoughHTTPRequestAttributes(t *testing.T) {
+	attributes := make(map[string]interface{})
+	attributes[semconventions.AttributeHTTPMethod] = "GET"
+	attributes[semconventions.AttributeHTTPScheme] = "http"
+	attributes[semconventions.AttributeHTTPClientIP] = "192.168.15.32"
+	attributes[semconventions.AttributeHTTPUserAgent] = "PostmanRuntime/7.21.0"
+	attributes[semconventions.AttributeHTTPTarget] = "/users/junit"
+	attributes[semconventions.AttributeHTTPHostPort] = 443
+	attributes[semconventions.AttributeNetPeerPort] = 8080
+	attributes[semconventions.AttributeHTTPStatusCode] = 200
```

Comment:
Can you add an assertion that status code was filled in?

Reply:
Yes great idea!